terragrunt/aws/s3/s3.tf

```diff
@@ -0,0 +1,11 @@
+module "s3_bucket" {
+  source = "github.com/cds-snc/terraform-modules//S3?ref=v10.4.1" # Using same version as OIDC module
```

Comment:
You want this bucket to be created in only the staging environment and not production. To do so, you can do 
  count = var.environment == "staging" ? 1 : 0
You don't have to do it now but once there is production deployments, you will have to.